core-bundle/src/Resources/contao/modules/ModuleBreadcrumb.php

```diff
@@ -199,7 +204,8 @@ protected function compile()
 			(
 				'isRoot'   => false,
 				'isActive' => true,
-				'href'     => $this->getPageFrontendUrl($pages[0]),
+				// Use the current request without query string for the current page (see #3450)
+				'href'     => strtok(Environment::get('request'), '?'),
```

Comment:
We should use the Symfony request here. `$request->getPathInfo()` might already have the path without the query string.

Reply:
It would be nice to use the Symfony request here.

However, I tried to keep the generated URL in line with the rest of the URLs in the breadcrumbs:
- The regular URLs contain the script name (e.g. `preview.php` or `app.php`) while `getPathInfo()` does not.
- The regular URLs are relative paths while `getPathInfo()` generates an absolute path reference (`page/subpage.html` vs. `/page/subpage.html`).

Whether the regular URLs are perfect surely is another good question.

I'm not quite convinced that manually building a fitting URL (trim leading `/`, add script name) is worth using the Symfony request:

````php
$request = System::getContainer()->get('request_stack')->getCurrentRequest();
ltrim($request->getScriptName() . $request->getPathInfo(), '/')

// or

strtok(Environment::get('request'), '?')
````

What do you prefer?